src/System.Reflection.Emit.ILGeneration/tests/Label/LabelEquals1.cs

```diff
@@ -77,5 +77,15 @@ public void PosTest6()
             Label label = new Label();
             Assert.False(label.Equals(ob));
         }
+
+        [Fact]
+        public void TestLabelEqualityOperators()
+        {
+            Label lb1 = new Label();
+            Label lb2 = new Label();
+
+            Assert.True(lb1 == lb2);
```

Comment:
Might want to add a user message if you use Assert.True/False, as otherwise it gives a pretty useless error message.